src/SciMLBenchmarks.jl

```diff
@@ -61,17 +69,20 @@ end
 
 function weave_folder(folder, build_list=(:script,:github))
   for file in readdir(folder)
-      # Skip non-`.jmd` files
-      if !endswith(file, ".jmd")
-          continue
-      end
+    # Skip non-`.jmd` files
+    if !endswith(file, ".jmd")
+      continue
+    end
 
-      try
-          weave_file(folder, file, build_list)
-      catch e
-          @show folder, file
-          @error(e)
+    try
+      @eval @subprocess begin
+        using SciMLBenchmarks
+        SciMLBenchmarks.weave_file($folder, $file, $build_list)
       end
+    catch e
+      @show folder, file
+      @error(e)
+    end
   end
 end
```

Comment:
@ChrisRackauckas What do you think about this? This prevents segfaults at least locally. All the subprocesses are blocking and carry over all the options so they won't interfere with the benchamrks

Reply:
That looks fine, just neds a rebase